docs/examples/word_embedding/word_embedding.md

```diff
@@ -119,8 +123,8 @@ len(vocab)
 ```
 
 By default, the vector of any token that is unknown to `vocab` is a zero vector.
-Its length is equal to the vector dimension of the fastText word embeddings:
-300.
+Its length is equal to the vector dimensions of the fastText word embeddings:
```

Comment:
Not sure which of these is more correct. Could also just say "...equal to the dimensionality/shape..."